tests/types/test_validators.py

```diff
@@ -0,0 +1,70 @@
+import strawberry
+
+
+def upper_validator(value, info):
+    return value.upper()
```

Comment:
I haven't seen the implementation yet, but I'm a bit confused on the terms, we are passing validators, but we are not doing any validation here 🤔

Reply:
I'll add some data validation.

As discussed in #788, this same method can be used for data pre/post processing and validation. Could we have more generic name for this? Or do we want to separate these so that there would be separate validator and processor? Any opinions?